src/olympia/devhub/utils.py

```diff
@@ -298,7 +298,8 @@ def extract_theme_properties(addon, channel):
         parsed_data = parse_xpi(
             version.file.file.path, addon=addon, user=core.get_user()
         )
-    except (ValidationError, ValueError):
+    except (ValidationError, ValueError) as exc:
+        log.exception('Error parsing xpi', exc_info=exc)
```

Comment:
I am wondering if this isn't going to send a lot of exceptions to Sentry.

Reply:
Would `log.debug` be more appropriate?

Reply:
It's only in `extract_theme_properties()` so it shouldn't - it's just for theme wizard uploads (which should always be valid)